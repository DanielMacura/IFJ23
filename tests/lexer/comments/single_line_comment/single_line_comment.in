    // begin example "double"
let x = 3.14    // Double
// end example