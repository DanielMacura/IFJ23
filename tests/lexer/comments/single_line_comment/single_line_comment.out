EOL
KEYWORD
IDENTIFIER
EQUALS
FLOAT:3.14
EOL
EOF
